test/test_triplets_classifiers.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from sklearn.exceptions import NotFittedError
+from sklearn.model_selection import train_test_split
+
+from test.test_utils import triplets_learners, ids_triplets_learners
+from sklearn.utils.testing import set_random_state
+from sklearn import clone
+import numpy as np
+
+
+@pytest.mark.parametrize('with_preprocessor', [True, False])
+@pytest.mark.parametrize('estimator, build_dataset', triplets_learners,
+                         ids=ids_triplets_learners)
+def test_predict_only_one_or_minus_one(estimator, build_dataset,
+                                       with_preprocessor):
+  """Test that all predicted values are either +1 or -1"""
+  input_data, preprocessor = build_dataset(with_preprocessor)
+  estimator = clone(estimator)
+  estimator.set_params(preprocessor=preprocessor)
+  set_random_state(estimator)
+  triplets_train, triplets_test = train_test_split(input_data)
+  estimator.fit(triplets_train)
+  predictions = estimator.predict(triplets_test)
+  for i in range(len(input_data)):
+    prediction = estimator.predict(input_data[None, i])
+    if(prediction == 0):
```

Comment:
drop extra parens

Reply:
actually, I forgot to take those lines out previously! I will erase them now.